packages/browser-wallet/src/shared/utils/number-helpers.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Given a list of numbers return the smallest unused non-negative integer.
```

Comment:
It might also be a precondition that there are no duplicate values in the array.

Reply:
Yeah, I guess it is, otherwise I would need to purge duplicates or adjust the condition. :thinking:

Reply:
Actually I don't? I tried testing it.

Reply:
But you didn't add a test for it hehe. Looking at it again I think it works with duplicates too, not that we should have them for the current usage.